src/org/opendatakit/aggregate/parser/BaseFormParserForJavaRosa.java

```diff
@@ -111,12 +116,29 @@ private static final void initializeJavaRosa() {
              // new CoreModelModule().registerModule();
              // replace with direct call to PrototypeManager
              PrototypeManager.registerPrototypes(SERIALIABLE_CLASSES);
+             redirectOutput();
```

Comment:
Since this call changes System.out and .err application wide, shouldn’t this call be made, in a very obvious way, at the top level of the application?